README.md

```diff
@@ -23,7 +23,7 @@ directly, via a `Timer`, or indirectly via a `Buffer`.
 ## Primary Use
 
 ```js
-import Igniter from 'igniter-core';
```

Comment:
I published this addon as `igniter-core` 0.0.0 already in order to claim the npm package (igniter was available when I began this but got sniped on me).  Is the change to `igniter-runtime` in order to keep consistent with `htmlbars-runtime` and `glimmer-runtime` ?  Regardless, assuming the package name is available I approve of the consistency and naming scheme.